budget_proj/budget_app/tests.py

```diff
@@ -28,6 +39,15 @@ def test(self):
 
         self.assertEqual(response.status_code, 200)
 
+    def test_history_get_request_works_with_query_param(self):
+        response = self.client.get("/budget/history/?fiscal_year=2015-16&bureau_code=PS")
+        self.assertEqual(response.status_code, 200)
+        json_content = json.loads(response.content.decode('utf-8'))
+        fiscal_years = [item["fiscal_year"] for item in json_content]
+
+        for fiscal_year in fiscal_years:
+            self.assertEqual(fiscal_year, 2015-16)
```

Comment:
LOL - wow, I must've had an awful Python instructor to make these kindergarten mistakes ;)